presto-main/src/main/java/com/facebook/presto/operator/HashAggregationOperator.java

```diff
@@ -309,7 +309,7 @@ private void processPage(Page page)
             }
         }
 
-        public boolean isFull()
+        public boolean checkFullAndUpdateMemory()
```

Comment:
Let's name this `updateMemoryUsage` and document that return value indicates whether it's full or not. We will split this into two functions later (so that we don't make changes beneath you).